testsuite/integration-arquillian/servers/auth-server/services/testsuite-providers/src/main/java/org/keycloak/testsuite/theme/TestThemeResourceProvider.java

```diff
@@ -1,11 +1,22 @@
 package org.keycloak.testsuite.theme;
 
+import org.keycloak.platform.Platform;
+import org.keycloak.provider.EnvironmentDependentProviderFactory;
 import org.keycloak.theme.ClasspathThemeResourceProviderFactory;
 
-public class TestThemeResourceProvider extends ClasspathThemeResourceProviderFactory {
+public class TestThemeResourceProvider extends ClasspathThemeResourceProviderFactory implements EnvironmentDependentProviderFactory {
 
     public TestThemeResourceProvider() {
         super("test-resources", TestThemeResourceProvider.class.getClassLoader());
     }
 
+    /**
+     * Quarkus detects theme resources automatically, so this provider should only be enabled on Undertow
+     *
+     * @return true if platform is Undertow
+     */
+    @Override
+    public boolean isSupported() {
```

Comment:
@pedroigor instead of having to re-build the testsuite-providers to exclude the class with a profile I'm just disabling it if the environment is not Undertow